### PR TITLE
Use drop instead of unlet

### PR DESCRIPTION
**Pull request by nounoursheureux**
Unlet isn't a real word, so it feels weird to use it to delete variables.

### CODE REVIEW

File: src/variables.rs [hunk 1]
```diff
@@ -61,7 +61,7 @@ impl Variables {
         SUCCESS
     }
 
-    pub fn unlet<I: IntoIterator>(&mut self, args: I) -> i32
+    pub fn drop<I: IntoIterator>(&mut self, args: I) -> i32
```
Comment (Ticki) [inline src/variables.rs]: FYI, this might conflict with `Drop`, if ever implemented.

**Comment by nounoursheureux** [comment]
@Ticki is it better with an underscore ?

**Comment by Ticki** [comment]
Maybe `drop_variable`. Underscores are generally hacky, and might even make the confusion worse.

**Comment by nounoursheureux** [comment]
@Ticki Agreed.

**Pull request merged**